experiences/top_articles/top_articles.test.js

```diff
@@ -0,0 +1,89 @@
+const { bootstrapExperience } = require("../../e2e/utils/bootstrap");
+const { scrollTo } = require("../../e2e/utils/scroll");
+const { touchCard } = require("../../e2e/utils/touch");
+const { isAtSnapPoint } = require("../../e2e/utils/snapPoints");
+const { isCardContentLoaded } = require("../../e2e/utils/cardContent");
+const { expect } = require("chai");
+const { getUrlFixture } = require("../../e2e/utils/fixtureUrl");
+const experience = require("./top_articles.json");
+
+describe("top_articles experience", function () {
+  let config, fixture;
+  const fixtureUrl = getUrlFixture({
+    siteUrl: "https://playground.marfeel.com/templates/article-skeleton.html",
+    requestHostname: "playground.marfeel.com",
+    technology: "web",
+    experienceUrl: "/experiences/top_articles/top_articles.json",
+  });
+
+  it("setup", async function () {
+    config = experience;
+    fixture = {
+      url: fixtureUrl,
+      articleTitle: "Article example",
+    };
+
+    await bootstrapExperience(browser, config, fixture);
+  });
+
+  it("card should render on scroll", async function () {
+    await scrollTo(browser, 400);
+
+    const firstCard = await browser.$(config.cards.topArticles.cardSelector);
+
+    const firstCardExists = await firstCard.waitForExist({ timeout: 5000 });
+
+    expect(firstCardExists).equal(true);
+  });
+
+  it("card should have right content", async function () {
+    const rightContentLoaded = await isCardContentLoaded(
+      browser,
+      config.cards.topArticles.cardSelector,
+      config.cards.topArticles.content
+    );
+
+    expect(rightContentLoaded).equal(true);
+  });
+
+  it("card should be displayed in viewport at initial snap point", async () => {
+    await scrollTo(browser, 800);
+
+    const firstCard = await browser.$(config.cards.topArticles.cardSelector);
+
+    const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
+
+    expect(firstCardIsInViewport).equal(true);
+
+    const isAtInitialSnapPoint = await isAtSnapPoint(
+      browser,
+      config.cards.topArticles.cardSelector,
+      config.cards.topArticles.snapPoints.initial
+    );
+
+    expect(isAtInitialSnapPoint).equal(true);
+  });
+
+  // fix minimise
```

Comment:
remove minimise until it works

Reply:
Do you mind if we open a PR for all of the tests once they are landed? it will be easier to find and replace once all of them are in the same branch. Now I opened PR's in different branches with this commented test